jupyter_resource_usage/api.py

```diff
@@ -75,6 +75,20 @@ async def get(self):
 
             metrics.update(cpu_percent=cpu_percent, cpu_count=cpu_count)
 
+        # Optionally get Disk information
+        if config.track_disk_usage:
+            try:
+                disk_info = psutil.disk_usage(config.disk_path)
+            except:
```

Comment:
What exceptions are being caught here? At a very minimum:

```suggestion
            except Exception:
```

so that this does not block the server extension from being killed with Ctrl+C interrupt